core/src/com/fallenflame/game/GameCanvas.java

```diff
@@ -6,6 +6,7 @@
 import com.badlogic.gdx.graphics.g2d.*;
 import com.badlogic.gdx.graphics.glutils.*;
 import com.badlogic.gdx.physics.box2d.*;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
```

Comment:
Damn you went for it, major props to you. I was highkey trying to avoid scene2d at all costs